cocos/animation/animation-clip.ts

```diff
@@ -384,6 +399,47 @@ export class AnimationClip extends Asset {
         return this._createEvalWithBinder(target, binder, context.rootMotion);
     }
 
+    public createAGEvaluation (context: AnimationClipGraphBindingContext) {
+        if (this._legacyDataDirty) {
+            this._legacyDataDirty = false;
+            this.syncLegacyData();
+        }
+
+        const trackEvaluations: AGTrackEvaluation<unknown>[] = [];
+        let exoticAnimationEvaluation: ExoticTrsAGEvaluation | undefined;
+
+        const { _tracks: tracks } = this;
+        const nTracks = tracks.length;
+        for (let iTrack = 0; iTrack < nTracks; ++iTrack) {
+            const track = tracks[iTrack];
+            if (track instanceof UntypedTrack) {
```

Comment:
This is not a breaking change: since the born of animation graph, it only accepts animations from model files.